core/geometry/src/polyface/PolyfaceQuery.ts

```diff
@@ -37,7 +37,7 @@ import { XYPointBuckets } from "./multiclip/XYPointBuckets";
 import { IndexedPolyface, Polyface, PolyfaceVisitor } from "./Polyface";
 import { PolyfaceBuilder } from "./PolyfaceBuilder";
 import { RangeLengthData } from "./RangeLengthData";
-
+/* eslint-disable-NotNow no-console */

```

Comment:
Should this and the commented off logging code be included in the PR?

Reply:
I left the on purpose.   It will probably be useful in the next round.  But not thereafter